src/Ethereum/Signer.cpp

```diff
@@ -87,7 +87,7 @@ Signature Signer::signatureDataToStructSimple(const Data& signature) noexcept {
     import_bits(r, signature.begin(), signature.begin() + 32);
     import_bits(s, signature.begin() + 32, signature.begin() + 64);
     import_bits(v, signature.begin() + 64, signature.begin() + 65);
-    return Signature{r, s, v};
+    return Signature{signature, r, s, v};
```

Comment:
Is this intended change?

Reply:
yeah, I didn't manage to convert r,s,v back to the signature, maybe @Milerius could help me there

Reply:
Let's add a helper function here: https://github.com/trustwallet/wallet-core/blob/588bd84642428db5fd417f41e1bac99e76128492/src/Ethereum/Signer.h#L51